pig/src/main/java/org/jboss/pnc/bacon/pig/impl/utils/CSVUtils.java

```diff
@@ -1,25 +1,25 @@
 package org.jboss.pnc.bacon.pig.impl.utils;
 
-import java.io.FileReader;
 import java.io.IOException;
 import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
 
 /**
  * @author Harsh Madhani<harshmadhani@gmail.com> Date: 06-August-2020
  */
 public class CSVUtils {
     public static Set<String> columnValues(String columnName, String filePath, char delimiter) throws IOException {
-        Reader in = new FileReader(filePath);
-        return CSVFormat.EXCEL.withDelimiter(delimiter)
-                .withFirstRecordAsHeader()
-                .parse(in)
-                .getRecords()
-                .stream()
-                .map(record -> record.get(columnName))
-                .collect(Collectors.toSet());
+        try (Reader in = Files.newBufferedReader(Paths.get(filePath), StandardCharsets.UTF_8);
+                CSVParser parser = CSVFormat.EXCEL.withDelimiter(delimiter).withFirstRecordAsHeader().parse(in)) {
+            return parser.getRecords().stream().map(record -> record.get(columnName)).collect(Collectors.toSet());
```

Comment:
@thescouser89 could we do something so that the autoformatter doesn't squash all the calls to a single line?

Reply:
My problem is with the `try`. Why is indenting like that when it breaks the two statements? I wish it would line it up all statements with the `try(`. It makes it really hard to read, IMO.

I thought that with a chain of calls like this, it already does align methods (on the dot `.`) if there's a certain number (or length). But again, I do not like how it aligns all methods except the first line which could have multiple method calls and is not aligned with the rest.

My only gripes (I think) with the current style is how it breaks lines on a open parenthesis and how it align lines. I think I was outvoted on the first, but the second issue may apply to that as well. Also, some cases may be unfixable due to how the formatter works.